### PR TITLE
fetch data from json file

### PR DESCRIPTION
**Pull request by Alexandru-Bulai**
# PR 21-03-24 : Fetch data
## Overview
In this PR we Fetch the data from out json file to display the images, price and name of the games
## Key Changes :
 1) indexDom.js : 
    - We retrieve img, name and price data for the game containers

Mar-21-2024 15-56-45

### CODE REVIEW

File: src/javaScript/games.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
       {
         "name": "The Witcher 3: Wild Hunt",
         "price": "40",
-        "img-cover": "the-witcher-3.png",
+        "imgCover": "the-witcher-3.png",
```
Comment (vladiiancu) [inline src/javaScript/games.json]: Why the change? It is not consistent with the other key names

File: src/javaScript/indexDom.js [hunk 2]
```diff
@@ -47,3 +47,32 @@ document.addEventListener('DOMContentLoaded', () => {
...
+  .then((response) => {
+    if (!response.ok) {
+      throw new Error('Network response was not ok')
+    }
+    return response.json()
+  })
+  .then((data) => {
+    const allGames = data.payload.gamesDetail
+    allGames.forEach((game) => {
+      const gamesContainer = document.querySelector('.grid-index')
```
Comment (vladiiancu) [inline src/javaScript/indexDom.js]: Do you think we need to query the selector in a loop? Or is it the same element so we can query it only once?

File: src/javaScript/games.json (hunk 1 above)
Comment (Alexandru-Bulai) [reply src/javaScript/games.json]: i changed all the key names in the end due to the fact that prettier adds a space between the `-` so when i try to use it come out like this `${game.img - cover}` and JS doesn't read the spaces in between

**Review by vladiiancu — APPROVED** [review]
